praktikumsplaner-frontend/frontend/src/components/MeldungPraktikumsstelle/MeldungOertlAusbilder.vue

```diff
@@ -0,0 +1,310 @@
+                            item-text="value"
+                            :rules="studiumsRule"
+                            prepend-icon="mdi-code-braces"
+                            filled
+                        >
+                        </v-select
+                    ></v-col>
+                    <v-col
+                        ><v-select
+                            v-model="praktikumsstelle.studiensemester"
+                            label="Studiensemester"
+                            :items="Studiensemester"
+                            item-value="name"
+                            item-text="value"
+                            :rules="studiumsRule"
+                            prepend-icon="mdi-calendar-range-outline"
+                            filled
+                        >
+                        </v-select
+                    ></v-col>
+                    <v-col
+                        ><v-select
+                            v-model="praktikumsstelle.studienart"
+                            label="Studienart"
+                            :items="Studienart"
+                            item-value="name"
+                            item-text="value"
+                            :rules="studiumsRule"
+                            prepend-icon="mdi-school-outline"
+                            filled
+                        >
+                        </v-select
+                    ></v-col>
+                </v-list-item>
+            </v-list>
+            <v-card-actions>
+                <v-spacer />
+                <v-btn
+                    text
+                    @click="cancel()"
+                >
+                    Abbrechen
+                </v-btn>
+                <v-btn
+                    color="primary"
+                    @click="uploadPraktikumsstelle()"
+                >
+                    Hochladen
+                </v-btn>
+            </v-card-actions>
+        </v-card>
+    </v-form>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { useSnackbarStore } from "@/stores/snackbar";
+import { Levels } from "@/api/error";
+import Praktikumsstelle from "@/types/Praktikumsstelle";
+import MeldungService from "@/api/MeldungService";
+import { useRules } from "@/composables/rules";
+import { Dringlichkeit } from "@/types/Dringlichkeit";
+import { YesNo } from "@/types/YesNo";
+import { Ausbildungsjahr } from "@/types/Ausbildungsjahr";
+import { Ausbildungsrichtung } from "@/types/Ausbildungsrichtung";
+import { Studiensemester } from "@/types/Studiensemester";
+import { Studienart } from "@/types/Studienart";
+import { Referat } from "@/types/Referat";
+import router from "@/router";
+
+const praktikumsstelle = ref<Praktikumsstelle>(
+    new Praktikumsstelle("", "", "", "", "")
+);
+const stuzubiSelection = ref<string>();
+const stuzubiSelectionItems = ref<string[]>(["Ausbildung", "Studium"]);
+const isAusbildung = ref<boolean>(false);
```

Comment:
Would it be possible to make this a computed property so we don't have to set the two values each time the selection changes and just return based on the selected value?